JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -121,6 +121,11 @@ public function normalize($object, $format = null, array $context = [])
             if ('id' === $attributeName || !$attributeMetadata->isReadable()) {
                 continue;
             }
+
+            if ($this->nameConverter) {
```

Comment:
You'r right, I miss the property accessor.

But, shouldn't it be moved after at line (https://github.com/jderusse/DunglasApiBundle/blob/fix-attributename-normalization/JsonLd/Serializer/ItemNormalizer.php#L129) ?

By moving at line 157, the normalisation will not convert colletions (https://github.com/jderusse/DunglasApiBundle/blob/fix-attributename-normalization/JsonLd/Serializer/ItemNormalizer.php#L145 and https://github.com/jderusse/DunglasApiBundle/blob/fix-attributename-normalization/JsonLd/Serializer/ItemNormalizer.php#L151)